elfpy/markets/hyperdrive/hyperdrive_market.py

```diff
@@ -107,8 +108,11 @@ def __setitem__(self, key, value):
     # fee percents
     trade_fee_percent: float = field(default=0.0)
     redemption_fee_percent: float = field(default=0.0)
+    governance_fee_percent: float = field(default=0.0)
 
-    # The amount of longs that are still open.
+    # governance fees that haven't been collected yet denominated in shares
+    gov_fees_accrued: Decimal = field(default=Decimal(0))
```

Comment:
Do the tests fail if this is a float? I'd like to keep it consistent with the other parameters and we will be getting rid of Decimal in a the upcoming precision PR.

Reply:
nope, changed it to float